katsdptelstate/telescope_state.py

```diff
@@ -62,16 +62,18 @@
 
 _INF = float('inf')
 
-_allow_pickle = True
-_warn_on_pickle = True
+_allow_pickle = False
+_warn_on_pickle = False
 _pickle_lock = threading.Lock()       # Protects _allow_pickle, _warn_on_pickle
 
 
 PICKLE_WARNING = ('The telescope state contains pickled values. This is a security risk, '
-                  'but is allowed because MeerKAT data up to March 2019 uses it. '
-                  'You can suppress this warning by setting KATSDPTELSTATE_ALLOW_PICKLE=1 '
-                  'in the environment, or disable pickles by setting '
-                  'KATSDPTELSTATE_ALLOW_PICKLE=0.')
+                  'but you have enabled it with set_allow_pickle.')
+PICKLE_ERROR = ('The telescope state contains pickled values. This is a security risk, '
+                'so is disabled by default. If you trust the source of the data, you '
+                'can allow the pickles to be loaded by setting '
+                'KATSDPTELSTATE_ALLOW_PICKLE=1 in the environment. This is needed for '
+                'MeerKAT data up to March 2019.')
 
 
 def set_allow_pickle(allow, warn):
```

Comment:
I would make the default value for `warn` False to simplify things for end users (like myself :-)) who can then just type `katsdptelstate.set_allow_pickle(True)`.